2021-22/2021-12-09_130p.py

```diff
@@ -0,0 +1,60 @@
+# 130 points - 60 lines - 30 seconds - 09.12.2021
+from vexcode_vrc import *
+from math import sqrt
+
+def main():
+    drivetrain.set_drive_velocity(100,PERCENT)
+    fork_motor_group.spin_to_position(1800, DEGREES, wait=False)
+    goto( -920,  920, 0)
+    goto( -920,-1450, 0)
+    goto(  750,-1400, 0) # blue in right zone
+    goto(  500,-1150, 1)
+#   goto(    0, -950, 0) # m = - 2/5
```

Comment:
Why is this commented line still in the code?

Reply:
It's the exact location to pick up the yellow goal. But with the adjusted corresponding coordinates the robot drives in a straight line, picking up the goal on the way.

Reply:
So m = - 2/5 is the slope or gradient of the linear motion?

Reply:
correct.